pkg/resource/cluster.go

```diff
@@ -18,23 +18,34 @@ package resource
 
 import (
 	"fmt"
+	"os"
+	"strings"
 
 	api "github.com/cockroachdb/cockroach-operator/api/v1alpha1"
+	"github.com/cockroachdb/cockroach-operator/pkg/clusterstatus"
 	"github.com/cockroachdb/cockroach-operator/pkg/condition"
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
+	RELATED_IMAGE       = "RELATED_IMAGE_COCKROACH_"
+	NotSupportedVersion = "not_supported_version"
+	CrdbContainerImage  = "crdb.io/containerimage"
```

Comment:
The convention I've seen in the k8s SDK is to include `Annotation` as a suffix for constants that are used as the keys for annotations.

Reply:
correct